src/main/java/ca/corefacility/bioinformatics/irida/ria/web/samples/dto/SampleConcatenationModel.java

```diff
@@ -0,0 +1,26 @@
+package ca.corefacility.bioinformatics.irida.ria.web.samples.dto;
+
+import java.util.List;
+
```

Comment:
JavaDoc

Reply:
Added in 2c94eb7d49c94c036f27427d7d02746f56a089d3